tests/tox.sh

```diff
@@ -89,20 +89,19 @@ CURRENT_CEPH_STABLE_RELEASE="$(echo $FLAVOR|awk -F ',' '{ print $1}')"
 # so we will build the desired CEPH_STABLE_RELEASE since the current patch didn't change the ceph version
 # Since we test all the Ceph releases, we will always test the impacted one
 if [[ "$CEPH_STABLE_RELEASE" != "$CURRENT_CEPH_STABLE_RELEASE" ]]; then
-  FLAVOR="$CEPH_STABLE_RELEASE,x86_64,ubuntu,16.04,_,ubuntu,16.04"
+  FLAVOR="$CEPH_STABLE_RELEASE,ubuntu,16.04"
 fi
 
 echo "Building flavor $FLAVOR"
-sudo make FLAVORS="$FLAVOR" build.parallel
-
-current_branch="$(git rev-parse --abbrev-ref HEAD)"
-# build tag from luminous,x86_64,centos,7,_,centos,7 to luminous-centos-7-x86_64
-TAG="${current_branch}-$(echo "$FLAVOR"|awk -F ',' '{ print $1"-"$3"-"$4"-"$2}')"
+make_output=$(sudo make FLAVORS="$FLAVOR" build.parallel)
```

Comment:
The problem I've seen with this approach is that when running the output is hidden in the variable, once done, the `-x` takes effects and show the content of the variable `make_output`. What's a bit troublesome is that if the command fails, you will get the error right away and then you will get the full output. This makes debugging a bit harder as we have to scroll all the way up to find the root cause.
Without the output inside a variable, you will get the error right at the end.

So now, I'm a bit hesitant with the new approach, what do you guys think?

Reply:
Good catch. We should not hide the build process just to catch a variable.
Two options: we can catch that variable out of the make build or we keep the actual code.

Reply:
To illustrate what I just said: https://paste.fedoraproject.org/paste/hxkygIjRXZmWKhWvjbITrQ

Reply:
We should parse the output so that if the format for tags changes in the future, we don't have to update this code again.

Reply:
Agreed but not on the build.parallel

Reply:
I will update this to fix.

Reply:
stage makes much more sense, perfect.